packages/components/carousel/src/carousel.vue

```diff
@@ -55,7 +57,7 @@
         @mouseenter="throttledIndicatorHover(index)"
         @click.stop="handleIndicatorClick(index)"
       >
-        <button :class="ns.e('button')">
+        <button :class="ns.e('button')" aria-label="Carousel Indicator">
```

Comment:
It would be better if you could give a hint as to which carousel to switch to.

Reply:
Done. Not sure whether the index should start from 0 or 1 here.